portfolio/src/main/java/com/google/sps/comment/Comment.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.comment;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+
+import java.io.IOException;
+
+public class Comment {
+    private String text;
+    private long timestamp;
+    private int score;
```

Comment:
lets add a javadoc to the attributes, explaining the score behavior (the range of the value, what does it means, etc)

Reply:
javadoc is usually put on top of the variable, not below it. and lets use // for single line comment
e.g
```
// The time the comment was submitted.
private long timestamp;
```

Reply:
`It ranges from 0 to 2, with 0 being the most negative and 2 being the most positive.`
Technically the value of the sentiment will be only either 1 or 2. it will never in between and never 0.
Maybe check the logic one more time to ensure that it does what you are thinking of?

Reply:
Yeah, I'm sorry, I was thinking of the original sentiment value, which is a float that ranges from -1 to 1. I floored the float and incremented it by 1 to get integers 0, 1, or 2. I will change my comment to rectify this mistake.